src/Build/Globbing/CompositeGlob.cs

```diff
@@ -40,13 +37,114 @@ public CompositeGlob(IEnumerable<IMSBuildGlob> globs)
         public CompositeGlob(params IMSBuildGlob[] globs) : this(globs.ToImmutableArray())
```

Comment:
ImmutableArray.Create(globs) would be the API to use here, as it avoids all the runtime checks for interfaces of the `IEnumerable<T>` overload

Reply:
Great point, and thanks &mdash; I'll use this in future. Again, I'll file a follow up PR that changes this.

This constructor is only used in test code in this repository. I would remove it, but it's public.